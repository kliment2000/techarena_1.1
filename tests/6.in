4
134170 14835720 4523930 2528312 
6
1 i 134170
2 i 340116
2 m 2468825
3 m 476794
3 k 134170
4 i 2528312
1
2 i
4
4 2 i m
4 3 i m
3 2 m m
1 3 i k
