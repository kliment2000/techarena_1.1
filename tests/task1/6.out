(1 ((2i 4 {2.m 4.i}) 3 {4.i 3.m} {2.m 3.m}) {1.i 3.k}) 47436528.9407099
